config/default/kustomization.yaml

```diff
@@ -32,7 +32,7 @@ patchesStrategicMerge:
 
 # Mount the controller config file for loading manager configurations
 # through a ComponentConfig type
-#- manager_config_patch.yaml
+- manager_config_patch.yaml
```

Comment:
It means that each time we update the config map we need to make sure to reboot the manager?

Reply:
That is correct. It's the same behavior as with the regular CLI configuration flags, but in the form of a `ConfigMap`.

Reply:
Thanks.